src/IdentityServer4/host/Startup.cs

```diff
@@ -96,12 +96,13 @@ public void Configure(IApplicationBuilder app)
 
             app.UseRouting();
             app.UseMiddleware<Logging.RequestLoggerMiddleware>();
-            app.UseIdentityServer();
 
+            app.UseCors();
```

Comment:
If you don't add the CORS middleware and try to invoke an endpoint that requires CORS, you'll get this big fat error:

![image](https://user-images.githubusercontent.com/582487/66520596-c252da00-eae9-11e9-99e6-d12e5f416ad6.png)